tools/hangar/__snapshots__/test_corpus/sdk_tests/dynamodb-table/aws-dynamodb.test.w_test_sim.md

```diff
@@ -1,5 +1,11 @@
 # [aws-dynamodb.test.w](../../../../../../examples/tests/sdk_tests/dynamodb-table/aws-dynamodb.test.w) | test | sim
 
+## stderr.log
+```log
+Error: No such object: amazon/dynamodb-local:2.0.0
```

Comment:
seems like we might want to swallow stderr when running `docker inspect`, no?

(we can do this later)

Reply:
yes, I've seen it also and I agree.

IMO, run command in general needs to be stronger (it disregard exit status)
opening a ticket